lib/oli_web/live/common/paged_table.ex

```diff
@@ -24,7 +24,7 @@ defmodule OliWeb.Common.PagedTable do
         {/if}
 
         {#if @total_count > 0 and @total_count > @limit}
-          <Paging id="header_paging" total_count={@total_count} offset={@offset} limit={@limit} click={@page_change}/>
+          <Paging id="header_paging" total_count={@total_count} offset={@offset} limit={@limit} click={@page_change} additional_class="bg-white"/>
```

Comment:
Can we avoid adding this class here? I think we may not want to change the background color of every paged_table the project renders.
I think having `<div class="mx-10 mb-10 bg-white">` in [this line](https://github.com/Simon-Initiative/oli-torus/blob/52eec16fa8ef7db223ed06955faef0487f55dc6b/lib/oli_web/components/delivery/learning_objectives/learning_objectives.ex#L58) would have the same result, but not affecting other places where we render the table.

Reply:
Nice catch 🙌